agent/app/agent_capability.go

```diff
@@ -457,8 +457,18 @@ func (agent *ecsAgent) appendServiceConnectCapabilities(capabilities []*ecs.Attr
 			return capabilities
 		}
 	}
-
-	return appendNameOnlyAttribute(capabilities, attributePrefix+capabilityServiceConnect)
+	loadedVer := agent.serviceconnectManager.GetLoadedAppnetVersion()
```

Comment:
Should we add a check here just in case `GetLoadedAppnetVersion()` returns  `nil `or an `empty` string?

Reply:
We have warning log on 463 and 464 which will log `loadedVer` if it's empty(one of the scenario).